decidim-core/app/controllers/decidim/account_controller.rb

```diff
@@ -56,6 +56,72 @@ def destroy
       redirect_to decidim.root_path
     end
 
+    def resend_confirmation_instructions
+      enforce_permission_to :update, :user, current_user: current_user
+
+      ResendConfirmationInstructions.call(current_user) do
+        on(:ok) do
+          respond_to do |format|
+            format.js do
+              render(
+                "resend_confirmation_email",
+                locals: {
+                  text: t("resend_successfully", scope: "decidim.account.email_change", unconfirmed_email: current_user.unconfirmed_email),
+                  alert_class: "success"
+                }
+              )
+            end
+          end
+        end
+
+        on(:invalid) do
+          respond_to do |format|
+            format.js do
+              render(
+                "resend_confirmation_email",
+                locals: {
+                  text: t("resend_error", scope: "decidim.account.email_change"),
+                  alert_class: "alert"
+                }
+              )
+            end
+          end
+        end
+      end
+    end
+
+    def cancel_email_change
+      enforce_permission_to :update, :user, current_user: current_user
+
+      if current_user.unconfirmed_email
+        current_user.update(unconfirmed_email: nil)
+
+        respond_to do |format|
+          format.js do
+            render(
+              "cancel_email_change",
+              locals: {
+                text: t("cancel_successfully", scope: "decidim.account.email_change"),
```

Comment:
Same also here, use instance variables `@text` and `@alert_class`.

Then, we can just call `render` without defining the template name (since it already matches with the action name).